rules/alerting.go

```diff
@@ -291,6 +291,13 @@ func (r *AlertingRule) SetRestored(restored bool) {
 	r.restored = restored
 }
 
+// Restored returns the if restoration state of the alerting rule.
+func (r *AlertingRule) Restored() bool {
```

Comment:
`GetRestored` aligns with other functions' naming.

```suggestion
// GetRestored returns the restoration state of the alerting rule.
func (r *AlertingRule) GetRestored() bool {
```

Reply:
https://github.com/prometheus/prometheus/blob/main/rules/alerting.go#L178-L203 i was follow convention here. But it's your call :)

Reply:
I was following convention at here: https://github.com/prometheus/prometheus/blob/main/rules/alerting.go#L178-L203. But it's your call :)

Reply:
The functions directly above it use "Get", which is what I had looked at: https://github.com/prometheus/prometheus/blob/d80e2e83c252272ee586a47edb906179caff3bfc/rules/alerting.go#L262-L287

But after referring to [Effective Go](https://golang.org/doc/effective_go#Getters), it seems that the proper way is to not use "Get", so I think this is good. Thanks!